src/foam/u2/tag/Select.js

```diff
@@ -64,6 +64,10 @@ foam.CLASS({
     {
       name: 'header',
       documentation: 'The heading text for the choices'
+    },
+    {
+      name: 'disabled_data',
+      documentation: 'Optional list of disabled choices that should not be selectable'
```

Comment:
After chatting with Sarthak, we are not sure if the list of disabled options should be a separate property like this or whether the bound data should have an additional array element for disabled status

Reply:
FYI basically for the MultiChoiceView have an array of selected items and an array of choices.